### DISCUSSION TITLE
[Question]: Docker-Installation hangs, Don't know how to fix it 😒

### DISCUSSION THREAD

**Discussion by simon-231** [danny-avila/LibreChat#2427] [Troubleshooting] [answered]
### What is your question?

I am trying to install LibreChat on my Ubuntu server using Docker, and everything proceeded smoothly until a specific step. I followed the Docker execution steps as described in the documentation. However, it seems to hang indefinitely at one point. and I'm unsure of how to proceed or where to look for solutions.

### More Details

I am using the instructions from https://docs.librechat.ai/install/installation/docker_compose_install.html but I run into problems when executing ...
```
# Start LibreChat
docker compose up
```

This is the moment, when docker stops to continue:
![image](https://github.com/danny-avila/LibreChat/assets/9551133/ceba76d7-7c8d-4730-973e-78e37d8475d7)

Any help or advice would be greatly appreciated!

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by fuegovic** [comment, marked as answer]
Have you tried accessing it at http://localhost:3080/ ?
![image](https://github.com/danny-avila/LibreChat/assets/32828263/ae59be0a-064a-4f9a-bbc8-f9efdbfa0878)

Usually when this pops up it means it is started and you can close the terminal window
(unless I missed an error in your screenshot?)

**Comment by aki-k** [comment]
I'm not really sure what problem simon-231 is having but in my case I can access the port 3080 through nginx reverse proxy and create the account, but after that I only get a spinning wheel in Firefox.

These are the changes I made to docker-compose.yml:
```
  api:
    ports:
      - "127.0.0.1:3080:3080/tcp"
```
```
  mongodb:
    image: mongo:4.4.6
```
The default mongodb docker image refused to work on my Core 2 Duo CPU. It complained about some missing CPU flag, it was maybe AVX.

**Comment by simon-231** [comment]
Now it works, thank you very much for your support and help. 

My problem was ultimately in two places:
1. I did not start the Docker container with "-d" and so it did not continue to run in the background.
2) I had set up a firewall on my Ubunut server and did not open the port for LibreChat, so I could not access the application.

Now that both problems have been solved, it works!
